style.css

```diff
@@ -0,0 +1,287 @@
+  width: 76px;
+  height: 20px;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: none;
+  flex-grow: 1;
+  border: rgb(247, 244, 244);
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+#bttn {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  padding: 12px;
+  background: #fff;
+  border: 1px solid #396df2;
+  border-radius: 12px;
+  margin: 15px 0;
+}
+
+#bttn:active {
+  background: #2e3de9;
+  color: #fff;
+}
+
+#bttn:hover {
+  background: #525ec4;
+  color: #fff;
+}
+
+.list {
+  list-style: none;
+  display: flex;
+  justify-content: stretch;
+  align-items: flex-end;
+  margin-bottom: 20px;
+}
+
+.canopy {
+  list-style: none;
+}
+
+a { text-decoration: none; }
+
+.text {
+  margin-top: 10px;
+  margin-left: -1px;
+  width: 55%;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  padding: 0;
+}
+
+.head-section {
+  width: 327px;
+  height: 146px;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.main-section {
+  align-items: center;
+  padding-left: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  padding-top: 10%;
+  padding-bottom: 10%;
+  line-height: 52px;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: repeat(5, auto);
+}
+
+.card_item {
+  border: 1px solid rgb(241, 234, 234);
+  border-radius: 10px;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-bottom: 24px;
+}
+
+.snap {
+  width: 295px;
+  height: 220px;
+  background: url(images/Snapshoot_Portfolio.png);
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.spa {
+  width: 295px;
+  height: 36px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-bottom: 24px;
+}
+
+.section6 {
+  width: 375px;
+  height: 1000px;
+  background: #fff;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  border: 1px solid rgb(241, 234, 234);
+  border-radius: 10px;
+}
+
```

Comment:
On this part look at the screenshot 

![image](https://user-images.githubusercontent.com/93630700/206737307-c849bc1f-082a-418b-91f4-b63a5a71e76a.png)

- [ ] Kindly add the fonts to the text corresponding to the FIGMA design 🥇 
- [ ] There's a space between missing on your subtitle after the multi-task text. Kindly fix it 💯

Reply:
all done